### ISSUE TITLE
`ncu` crashes when root `package.json` contains `workspaces`  → `packages`

### ISSUE THREAD

**Issue by kachkaev** [raineorshine/npm-check-updates#1214]
👋 @raineorshine!

- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates` (16.3.16)
- [x] I am using `node >= 14.14`

---

## Steps to Reproduce

.ncurc:

n/a

Dependencies:

```json
{
  "workspaces": {
    "packages": ["packages/*"],
    "nohoist": ["**/react-native", "**/react-native/**"]
  }
}
```

Source: https://classic.yarnpkg.com/blog/2018/02/15/nohoist/

Steps:

1. Run `ncu --workspaces`

## Current Behavior

`ncu` crashes

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received an instance of Object
    at new NodeError (node:internal/errors:387:5)
    at validateString (node:internal/validators:162:11)
    at Object.join (node:path:1172:7)
    at /Users/ak/.nvm/versions/node/v16.18.0/lib/node_modules/npm-check-updates/build/src/index.js:249:50
    at Array.map (<anonymous>)
    at runUpgrades (/Users/ak/.nvm/versions/node/v16.18.0/lib/node_modules/npm-check-updates/build/src/index.js:249:18) {
  code: 'ERR_INVALID_ARG_TYPE'
```

## Expected Behavior

`ncu` works for `workspaces` → `packages` → `[]` as for `workspaces` → `[]`

```
{
  "workspaces":  ["packages/*"]
}
```

## Relevant  code

https://github.com/raineorshine/npm-check-updates/blob/6aaf11f053d915d49aad75d3ba121b280b9b724e/src/index.ts#L220-L221

https://github.com/raineorshine/npm-check-updates/blob/6aaf11f053d915d49aad75d3ba121b280b9b724e/src/index.ts#L254-L255

**Comment by raineorshine** [comment]
Thanks for reporting!

Fixed and published in `v16.3.17`.

**Comment by kachkaev** [comment]
Big thanks @raineorshine! I just upgraded `npm-check-updates` to `16.3.17` and it worked 🔥
